### ISSUE TITLE
SA1008 with unpacking tuple is not fixed in 1.1.118

### ISSUE THREAD

**Issue by ChrisTorng** [DotNetAnalyzers/StyleCopAnalyzers#2952]
According to #2898, the unpacking tuple problem should be resolved in 1.1.x, but it's not.

`var (variableA, variableB) = MyMethodThatReturnsATuple();`
has "SA1008 Opening parenthesis must not be preceded by a space.".

If I change it into:
`var(variableA, variableB) = MyMethodThatReturnsATuple();`
It has "SA1008 Opening parenthesis should be preceded by a space.", plus two more: "SA1000 The keyword 'var' should be followed by a space." and "IDE0055 Fix formatting (The keyword 'var' should be followed by a space.)". The suggestion fix is adding back the space.

`public (MyType, MyOtherType) MyMethodThatReturnsATuple()`
has "SA1008 Opening parenthesis must not be preceded by a space.".

If I change it into:
`public(MyType, MyOtherType) MyMethodThatReturnsATuple()`
It has "SA1008 Opening parenthesis should be preceded by a space.", plus one more:  "IDE0055 Fix formatting". The suggestion fix is adding back the space.

**Comment by sharwell** [comment]
I am not able to reproduce either of these issues with 1.1.1-rc.114 or 1.1.118. The follow method compiled without reporting SA1008:

```csharp
private (AccessViolationException, ActivationContext) MethodThatReturnsTuple()
{
    var (a, b) = this.MethodThatReturnsTuple();
    throw null;
}
```

**Comment by ChrisTorng** [comment]
I found the problem in my .NET Framework project. There are two different StyleCop.Analyzers references in csproj:

    <PackageReference Include="StyleCop.Analyzers">
      <Version>1.1.118</Version>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
      <PrivateAssets>all</PrivateAssets>
    </PackageReference>

    <Analyzer Include="..\..\..\..\ThirdParty\NuGetPackage\StyleCop.Analyzers.1.0.2\analyzers\dotnet\cs\StyleCop.Analyzers.CodeFixes.dll" />
    <Analyzer Include="..\..\..\..\ThirdParty\NuGetPackage\StyleCop.Analyzers.1.0.2\analyzers\dotnet\cs\StyleCop.Analyzers.dll" />

I can see two different versions of StyleCop.Analyzers under "References - Analyzers". After removing all of them, then use NuGet to get newest StyleCop.Analyzers, the `<Analyzer Include=` one was deleted. Then SA1008 is not reporting again.

As I remembered, this csproj had [Migrate from packages.config to PackageReference](https://docs.microsoft.com/en-us/nuget/reference/migrate-packages-config-to-package-reference) before. But I don't remember the exact steps I took. I think I should remove the `<Analyzer Include=` while in migrating to avoid this kind of problem.

Thanks for your kindly help. Maybe the installing procedure of StyleCop.Analyzers can detect this situation/problem and delete `<Analyzer Include=` automatically?
